Table.html

```diff
@@ -2,81 +2,8 @@
 <html>
 
 <head>
+    <link rel="stylesheet" href="table.css">
     <title>Table</title>
-    <style>
-        body{
-            font-family: sans-serif;
-            background-color: aliceblue;
-        }
-        section {
-            position: relative;
```

Comment:
Hello,

1) Create a folder css-table and put the HTML and CSS file in that.
2) The approach you have taken to make the head fix can be simplify. All you need to do is remove postion from main section and divs in th. Make the thead fixed only.
3) You have declared the background color to the main section. Again it is not required, you can set that too the thead.
4) Hover effect should be on whole row.

Reply:
Hi,
ok , let me do that changes .

Reply:
Hi,
   I have Done with the all the above changes , please review it once .